src/ui/pages/Backup/useRecoveryPhrase.ts

```diff
@@ -0,0 +1,26 @@
+import { useQuery } from '@tanstack/react-query';
+import { invariant } from 'src/shared/invariant';
+import { walletPort } from 'src/ui/shared/channels';
+
+export function useRecoveryPhrase({ groupId }: { groupId?: string }) {
+  return useQuery({
+    queryKey: ['getRecoveryPhrase', groupId],
+    queryFn: async () => {
+      if (groupId) {
+        invariant(groupId, 'groupId param is required for wallet appMode');
```

Comment:
why `invariant` if already checked for truthyness?

Reply:
thats my refactoring madness, thanks